### DISCUSSION TITLE
High availability?

### DISCUSSION THREAD

**Discussion by johnsonjsyuen** [supabase/supabase#1504] [General]
Can you talk a little about high availability? Is it implemented? Will we know about fail over?

**Comment by johnsonjsyuen** [comment]
No response after a year On a fundamental question?

**Reply by kiwicopple** [reply]
Hey @johnsonjsyuen - I'm adding a timestamp because this information will be outdated in the future: 5 July 2022

This is available on the Enterprise tier (reach out [here](https://supabase.com/contact/enterprise)). HA is currently not available on our Free/Pro tier, and it's the main reason why we have kept the "Beta" status. It shouldn't be far away, although I can't give a concrete date yet.

**Reply by johnsonjsyuen** [reply]
thank you much appreciated! "Beta" as in it may come to Pro tier in the future?

**Reply by kiwicopple** [reply]
Yup!

**Comment by bennichols** [comment]
What's the status of this @kiwicopple? I can't find anything else about it and this hasn't been updated for two and a half years.

**Comment by craft365** [comment]
@bennichols It seems like you search the information provided in this srticle:
https://supabase.com/ga

**Reply by bennichols** [reply]
Hi @craft365, there is literally no mention of high availability in that article.

**Comment by LS-Mochi** [comment]
any update on this?
